prov/slurm/execution.go

```diff
@@ -404,8 +402,8 @@ func (e *executionCommon) buildJobOpts() string {
 		opts += fmt.Sprintf(" --ntasks=%d", e.jobInfo.Tasks)
 	}
 	opts += fmt.Sprintf(" --nodes=%d", e.jobInfo.Nodes)
-	if e.jobInfo.Mem != 0 {
-		opts += fmt.Sprintf(" --mem=%dG", e.jobInfo.Mem)
+	if e.jobInfo.Mem != "" {
+		opts += fmt.Sprintf(" --mem=%s", e.jobInfo.Mem)
```

Comment:
I think using a `scalar-unit.size` will prevent using malformed values.
What do you think? 
That said, maybe a it will be necessary to canonize it using go-humanize for instance.

Reply:
But I am ok it's more elegant and user-friendly in UI

Reply:
Cool :smile: